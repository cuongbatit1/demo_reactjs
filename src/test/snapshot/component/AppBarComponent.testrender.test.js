import { render, screen } from "@testing-library/react";
import AppBarComponent from "../../../component/AppBarComponent";

test("test renders AppBarComponent snapshot", () => {
  const { asFragment } = render(<AppBarComponent />);
  expect(asFragment()).toMatchSnapshot();
});

test("test renders AppBarComponent snapshot with open true", () => {
  const { asFragment } = render(<AppBarComponent open={true} />);
  expect(asFragment()).toMatchSnapshot();
});
