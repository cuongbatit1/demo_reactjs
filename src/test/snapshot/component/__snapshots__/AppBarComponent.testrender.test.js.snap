// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders AppBarComponent snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-19z1ozs-MuiPaper-root-MuiAppBar-root"
  />
</DocumentFragment>
`;

exports[`test renders AppBarComponent snapshot with open true 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-1191obr-MuiPaper-root-MuiAppBar-root"
  />
</DocumentFragment>
`;
